recipe/meta.yaml

```diff
@@ -20,18 +20,22 @@ requirements:
     - python >=3.8
     - pytest-runner >=5.2
   run:
-    - aicsimageio >=4.1
+    - aicsimageio >=4.6.3
     - fsspec
-    - napari >=0.4.11
-    - napari-plugin-engine >=0.1.4
+    - napari >=0.4.13
+    - psutil >=5.7.0
     - python >=3.8
+    # plugins
+    - bioformats_jar >=2020.5.27
+    - readlif >=0.6.4
+    - nd2 >=0.3.0
```

Comment:
do we need `[legacy]`? https://github.com/AllenCellModeling/aicsimageio/blob/main/setup.py#L37

Reply:
since conda doesnt have the concept of extras, I [added it by default](https://github.com/conda-forge/nd2-feedstock/blob/1269c7b6b5e9646405e015fa294e87bc8d552566/recipe/meta.yaml#L37-L38) to the nd2 feedstock